lib/pmdtester/builders/diff_report/configerrors.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Contains methods to write out html for the configuration errors.
+# This mixin is used by DiffReportBuilder.
+module DiffReportBuilderConfigErrors
+  def build_configerrors_section(doc, configerrors_diffs)
+    doc.div(class: 'section', id: 'configerrors') do
+      doc.h2 'Configuration Errors:'
+
+      doc.h3 PmdTester::HtmlReportBuilder::NO_DIFFERENCES_MESSAGE if configerrors_diffs.empty?
+      configerrors_diffs.each do |key, value|
+        doc.div(class: 'section') do
+          doc.h3 key
+          build_configerrors_table(doc, value)
+        end
+      end
+    end
+  end
+
+  def build_configerrors_table(doc, errors)
+    doc.table(class: 'bodyTable', border: '0') do
+      build_configerrors_table_head(doc)
+      build_configerrors_table_body(doc, errors)
+    end
+  end
+
+  def build_configerrors_table_head(doc)
```

Comment:
The structure of this method is similar to [build_errors_table_head](https://github.com/pmd/pmd-regression-tester/pull/59/files#diff-eb2dc6eb4abed74d6a76c2d3befa8a9fR27). Maybe we should define a method named `build_table_head(doc, head1, head2)` in `HtmlReportBuilder ` class. And the method should be:
```
def build_configerrors_table_head(doc)
    build_table_head(doc, 'Rule', 'Message')
end
```
`build_errors_table_head(doc)` should be:
```
def build_errors_table_head(doc)
    build_table_head(doc, 'Message', 'Details')
end
```

Reply:
Done.